modules/adding-worker-nodes.adoc

```diff
@@ -0,0 +1,36 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
+
+:_module-type: CONCEPT
+[id="adding-worker-nodes_{context}"]
+= Adding worker nodes
+
+After you deploy your OpenShift Container Platform cluster, you can add worker nodes to scale cluster resources. There are different ways you can add worker nodes depending on the installation method and the environment of your cluster.
+
+* For installer-provisioned infrastructure (IPI) clusters, you can manually or automatically scale the machineset to match the number of available bare metal hosts.
++ 
+To add a new bare metal host, you must configure all network prerequistites, configure an assoicated `baremetalhost` object, then provision the worker node to the cluster. You can add a new bare metal host manually or by using the web console. 
```

Comment:
s/assoicated/associated

Reply:
@elfosardo - hey - I'm looking to get a LGTM on this PR so I can move it forward. Are you able to give a LGTM on some or all of the content? I know some might be outside of your remit. It's a bit tricky as it crosses teams.